exercises/formatSeconds/formatSeconds.js

```diff
@@ -21,6 +21,24 @@ function formatSeconds(num) {
 
   // Remember, if the code is stumping you, take a step back and
   // make sure you can do it by hand.
+
+  if (num < 60) {
```

Comment:
@keithjng How would you do this by hand? I don't think you're start with seconds. I think you'd ask "How many weeks fit into this number of seconds?" "How many days fit into what's leftover after we fit in weeks?" "How many hours fit into what's leftover after we fit in days?" etc.

```javascript
let SECONDS_IN_MINUTE = 60;
let SECONDS_IN_HOUR = SECONDS_IN_MINUTE * 60;
let SECONDS_IN_DAY = SECONDS_IN_HOUR * 24;

let remainingSeconds = num;

let numDays = Math.floor(remainingSeconds / SECONDS_IN_DAY);
remainingSeconds = remainingSeconds % SECONDS_IN_DAY;

let numHours = Math.floor(remainingSeconds / SECONDS_IN_HOUR);
remainingSeconds = remainingSeconds % SECONDS_IN_HOUR;

let numMinutes = Math.floor(remainingSeconds / SECONDS_IN_MINUTE);
remainingSeconds = remainingSeconds % SECONDS_IN_MINUTE;
```

We have to use `Math.floor` because JavaScript doesn't have integer division.